build/README-DB2.md

```diff
@@ -11,15 +11,18 @@ Running fhir server integration tests using docker db2.
 
 ## Run
 
-From the root of the project (aka the "WORKSPACE"):
-1. Set up and start the containers
-    ```sh
-    build/pre-integration-test-docker.sh
+From the root of the project (aka the "WORKSPACE"):  
+1. Set up and start the containers  
+    ```sh  
+    build/pre-integration-test-docker.sh  
     ```
-2. Run the tests
+    
+   Note: If you are testing NATS notifications, invoke the NATS subscriber via `node fhir-server-test/src/test/nodejs/nats-subscriber`.  If this is your first time, install the dependencies first by installing [Node.js](https://nodejs.org/en/download) (if not already installed) and running `(cd fhir-server-test/src/test/nodejs && npm install)`.  
```

Comment:
`(` is intentional?  also matching `)`

Reply:
yes, its to change the dir for just the scope of the single sub-command

Reply:
alternatively, if read as a parenthetical, the user can just cd to that dir to install the client, then change back